features/rails_features/integrations.feature

```diff
@@ -1,14 +1,10 @@
 Feature: App type is set correctly for integrations in a Rails app
 
-Background:
-  Given I start the rails service
-  And I run the "db:prepare" rake task in the rails app
-  And I run the "db:migrate" rake task in the rails app
```

Comment:
This background no longer works as the environment variable `ACTIVE_JOB_QUEUE_ADAPTER` is set too late now — `docker compose exec` re-uses the already running container, so environment variables must be set before starting the container

Reply:
Most of these tests didn't need the database setup anyway — only Que & Delayed Job rely on it